src/NuGet.Core/NuGet.PackageManagement/Utility/JsonConfigUtility.cs

```diff
@@ -173,6 +214,14 @@ private static JObject SortProperties(JObject parent)
             return sortedParent;
         }
 
+        private static bool HasFramework(IEnumerable<NuGetFramework> list, NuGetFramework framework)
+        {
+            return list.Any(
+                item =>
+                    string.Equals(item.Framework, framework.Framework, StringComparison.OrdinalIgnoreCase) 
```

Comment:
`NuGetFramework` employs its own comparer, so `list.Any(framework)` should just work.